pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -58,6 +60,22 @@ as the protocol string.
 
 ## Protocol extensions
 
+### Direct Peering Agreements
```

Comment:
```suggestion
### Explicit Peering Agreements
```

When preparing the presentation for the Matrix, I found it simpler to explain by calling these "Explicit" to contrast with the "Peering Agreements" that are created through GRAFTing

Reply:
Well, the explicit is slightly different -- they are direct peers in the protocol, hence direct peering agreements.

Reply:
But they are set by the user/app, hence explicit.

All the peering agreements are always between two nodes, hence one could call all of them direct.

Reply:
Precisely! These are agreements between pairs of peers, so they are called direct peering agreements in the protocol.

Reply:
So the contrast is between "Mesh peering agreements" and "direct peering agreements".

Reply:
At any rate I don't feel all that strongly about the wording in the section title; if you insist I will bulge :)

Reply:
we resolved async, discussed the merits of both terms and ended up with "explicit. Next commit updates the spec, including doctoc :)